src/query-builder/SelectQueryBuilder.ts

```diff
@@ -2025,7 +2026,13 @@ export class SelectQueryBuilder<Entity> extends QueryBuilder<Entity> implements
             // transform raw results into entities
             const rawRelationIdResults = await relationIdLoader.load(rawResults);
             const rawRelationCountResults = await relationCountLoader.load(rawResults);
-            const transformer = new RawSqlResultsToEntityTransformer(this.expressionMap, this.connection.driver, rawRelationIdResults, rawRelationCountResults, this.queryRunner);
+
+            let transformer;
+            if(this.connection.driver instanceof OracleDriver) {
+                transformer = new OracleRawSqlResultsToEntityTransformer(this.expressionMap, this.connection.driver, rawRelationIdResults, rawRelationCountResults, this.queryRunner);
+            } else {
```

Comment:
Isn't this exactly the same except everything is uppercase?  Couldn't this be achieved by calling a function on the driver to normalize the column name

Reply:
It is, yep - Couldn't see another way right now and intention - initially - to avoid messing with the RawSqlResultsToEntityTransformer and bring a wider range of drivers in to scope but happy to relook and see if I can find a better way with more time to check